src/test/java/net/sf/jabref/logic/groups/GroupsUtilTest.java

```diff
@@ -28,8 +28,7 @@ public void test() throws IOException {
         try (BufferedReader fr = Files.newBufferedReader(Paths.get("src/test/resources/testbib/testjabref.bib"),
                 StandardCharsets.UTF_8)) {
 
-            ParserResult result = BibtexParser.parse(fr,
-                    JabRefPreferences.getInstance().getImportFormatPreferences());
+            ParserResult result = BibtexParser.parse(fr, JabRefPreferences.getInstance().getImportFormatPreferences());
```

Comment:
Same here (although I could have rerun the same, was traveling and then ran out of time).